src/TypeScript/Sarif.SDK/package.json

```diff
@@ -30,7 +30,7 @@
     "@types/chai": "^4.1.3",
     "@types/mocha": "^5.2.0",
     "chai": "^4.1.2",
-    "mocha": "^5.1.1",
+    "mocha": "^9.2.2",
```

Comment:
>mocha": "^9.2.2 [](http://example.com/codeflow?start=5&length=15)

Hi @jeffersonking / @EasyRhinoMSFT,

do you know how can I test this?

Reply:
Should be `npx mocha`.